commands/promote.js

```diff
@@ -17,9 +17,17 @@ function * run (context, heroku) {
     // If no current DATABASE, return nil.
     let attachments = yield heroku.get(`/apps/${app}/addon-attachments`)
     let current = attachments.find(a => a.name === 'DATABASE')
+    console.log(current)
+    console.log(attachment)
```

Comment:
Can you drop the debugging statements (or change them to use `debug`)?

Reply:
removed, i had forgotten to remove those at the end of development